src/renderer/components/basic.lua

```diff
@@ -51,7 +51,14 @@ function basicTextComponent:render(surf, position, styles, resolver)
   end
 
   local cY = position.top + topPad
-
+  
+  local totalSize = 1;
+  local parent = self;
+  while (parent ~= null) do
+    totalSize = totalSize * tonumber(parent.styles["font-size"]:substring(1,1));
```

Comment:
It can be longer than one character, what about `0.5em` or `12em`. Use `string.match` instead.

Reply:
Eh, I felt that It'd leave the substring for when we actually make it work in issue #6